src/main/java/hudson/plugins/dimensionsscm/CallbackInstance.java

```diff
@@ -37,13 +37,12 @@ private static boolean supportsChangeSets(DimensionsConnection connection) {
 
     /*We need only number part of server version*/
     private static String normalizeServerVersion(String version) {
-        Pattern regPattern = Pattern.compile("([\\d]+)(.*)");
+        Pattern regPattern = Pattern.compile("([\\d]+)(.*|R*)");
```

Comment:
What does this do? `R*` in a regular expression will match R, RR, RRR, etc. I think this change is not needed.

Reply:
Sorry, my mistake.
I deleted it and push.